packages/manager/apps/hub/src/index.less

```diff
@@ -2,7 +2,8 @@
 @import '@ovh-ux/manager-account-sidebar/src/variables.less';
 
 .hub-main-view_sidebar_expanded {
-  margin-right: @ovh-sidebar-width;
+  // -0.75 rem is to take the scrollbar width into account
+  margin-right: calc(@ovh-sidebar-width - 0.75rem);
```

Comment:
well maybe it could be in a `@scrollbar-width` then, it looks like a value we could reuse :)